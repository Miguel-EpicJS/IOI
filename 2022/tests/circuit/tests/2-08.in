385045b28fbc1903eca997d513040235e0229e01
999 1000 5
-1 0 1 2 3 0 3 6 5 8 6 8 11 9 1 14 10 12 10 13 2 20 4 17 22 13 12 26 15 11 14 27 31 4 30 19 32 18 32 21 38 36 30 19 17 25 23 25 39 27 48 40 41 41 47 38 46 56 54 56 34 45 20 52 57 58 62 65 65 29 40 43 28 54 5 34 26 15 61 67 49 9 43 46 73 76 68 79 87 83 81 47 82 92 68 52 28 86 69 60 93 71 71 82 87 31 60 69 79 63 92 98 64 36 94 113 105 111 91 67 115 90 77 44 58 64 74 95 62 100 110 121 130 129 117 126 49 95 130 80 91 127 72 107 140 98 81 105 126 45 128 135 22 140 151 152 88 80 89 146 73 100 134 146 145 145 138 137 121 107 153 157 90 61 159 149 111 125 139 170 163 173 83 161 154 115 117 150 165 152 160 66 190 177 143 110 164 55 186 138 178 29 176 119 76 163 188 141 202 55 195 204 137 149 201 154 215 132 173 153 147 180 168 213 195 203 199 120 167 134 96 226 188 164 21 72 183 215 214 141 148 132 205 167 204 74 128 207 246 224 135 229 192 151 165 172 53 148 234 216 114 201 214 256 210 160 225 125 182 242 222 179 206 272 235 240 183 256 258 270 118 253 262 216 255 159 237 263 176 268 205 258 229 253 144 257 191 261 181 186 285 209 255 198 208 289 219 283 266 222 224 275 270 301 291 310 287 139 174 295 302 292 252 265 317 306 249 178 119 318 210 285 245 289 328 310 238 238 174 282 254 249 271 293 263 277 257 196 338 297 295 343 288 245 349 275 345 306 301 351 350 284 355 336 262 353 357 336 180 254 93 276 354 202 370 219 359 129 206 347 357 380 182 376 370 286 279 291 364 302 352 386 363 344 260 367 369 233 338 312 362 198 332 341 280 391 382 272 382 404 408 375 246 308 365 395 320 388 416 359 355 380 326 343 369 322 329 281 365 402 419 317 389 309 250 364 398 389 303 318 439 436 414 332 297 393 442 363 292 347 329 395 279 440 415 284 325 288 454 452 433 337 443 265 406 424 404 449 387 390 461 468 414 466 367 240 474 419 477 472 426 405 268 411 261 361 477 353 349 408 446 447 475 444 443 423 434 439 460 426 487 462 488 293 474 493 371 406 430 430 478 351 346 505 415 448 455 484 484 505 496 510 435 341 512 118 475 500 471 524 409 478 368 532 494 496 524 455 375 390 444 424 525 517 387 527 516 491 490 542 335 434 379 542 493 532 526 550 373 558 466 500 558 361 537 562 250 564 394 510 459 543 550 563 379 512 479 420 299 575 511 549 518 543 393 501 485 234 449 559 518 469 447 591 534 535 516 346 519 563 488 479 491 520 554 597 480 432 507 352 590 547 546 528 554 601 612 582 545 581 467 605 501 513 566 432 622 551 540 541 614 595 627 627 591 429 525 551 503 588 633 617 572 589 620 634 405 643 623 579 586 511 592 620 469 545 607 605 549 613 517 641 592 595 636 555 403 660 611 471 610 597 664 663 610 565 633 583 596 614 581 651 559 613 598 624 678 637 340 664 669 539 676 687 682 669 461 472 411 594 643 539 632 662 656 688 598 402 577 533 580 694 698 707 670 667 694 637 682 337 699 587 494 674 702 719 656 606 634 593 528 628 725 535 566 594 733 588 575 721 585 707 725 705 739 726 722 655 640 723 632 745 561 636 746 446 695 638 747 699 607 687 196 731 547 677 755 674 761 673 467 638 659 757 729 653 733 671 587 617 684 772 743 757 651 750 748 734 759 630 761 604 770 787 792 688 730 599 778 773 776 777 718 487 801 623 793 578 724 747 803 789 753 789 724 800 808 744 758 606 630 770 741 777 710 771 797 811 750 803 744 771 791 795 769 814 582 740 815 534 755 489 676 832 779 801 503 483 769 719 846 826 826 833 768 730 836 798 845 858 713 819 726 840 702 795 743 812 527 846 868 783 734 815 873 834 762 722 829 834 822 863 776 879 822 808 845 783 625 880 873 871 883 708 820 850 762 880 713 842 800 690 883 900 838 896 746 590 905 868 856 900 850 836 857 625 864 877 876 911 787 823 862 779 685 896 663 721 872 752 869 906 829 877 915 931 907 919 729 930 931 913 926 911 781 943 930 919 947 946 929 915 906 935 946 907 882 947 926 912 952 949 821 960 927 934 956 941 943 952 920 934 918 740 972 921 968 961 971 870 864 956 890 976 942 904 935 981 904 939 971 981 968 976 980 994 980 995 992 7 16 18 23 24 33 37 37 33 42 39 24 50 35 42 44 59 57 7 63 51 86 85 85 48 75 103 108 75 59 78 99 77 124 89 51 78 123 120 112 104 101 106 50 114 113 131 142 94 70 109 131 102 150 123 127 53 122 155 109 143 133 66 124 162 35 168 97 169 162 16 96 158 171 156 88 70 147 116 191 187 84 101 169 102 166 171 193 161 170 177 197 144 175 194 133 122 197 187 106 194 212 104 142 200 217 208 156 193 112 155 189 136 185 227 230 136 211 236 84 228 217 211 190 239 239 103 247 243 184 199 241 242 244 228 158 184 212 97 248 225 175 264 269 267 247 243 235 116 203 185 166 189 220 231 278 252 108 290 287 237 226 221 200 99 266 233 172 267 220 286 313 181 303 274 296 307 305 223 304 269 264 236 283 304 157 259 298 231 322 330 276 331 324 327 230 316 244 259 278 280 298 300 294 273 330 223 321 314 358 227 325 314 348 319 307 277 358 319 300 335 372 383 260 221 294 394 348 333 311 209 271 281 397 366 315 342 397 323 378 398 324 377 248 384 328 396 327 312 374 282 218 345 377 305 413 356 416 232 290 385 331 400 371 296 384 386 385 356 407 378 273 241 374 192 401 399 334 421 311 440 448 427 418 388 400 399 213 422 360 316 376 418 450 420 454 409 251 344 428 435 392 442 451 410 463 372 470 421 427 465 350 313 274 476 459 396 401 473 480 438 468 453 481 457 428 179 392 407 465 470 453 431 498 485 333 502 362 483 464 381 368 339 456 373 438 437 417 499 433 481 492 412 482 482 445 360 507 251 508 403 499 422 492 519 339 366 506 354 513 299 413 431 495 486 436 497 437 486 320 495 308 456 460 489 464 521 541 504 490 458 417 498 218 538 451 515 321 523 555 529 521 452 523 326 520 556 476 423 425 544 207 568 544 381 538 562 560 309 522 570 571 557 429 569 506 567 412 514 571 548 553 450 556 531 546 458 589 533 593 599 560 601 572 585 608 425 583 564 536 574 502 536 619 604 622 522 631 586 618 642 603 629 548 647 565 640 641 602 567 648 315 557 540 635 508 577 629 616 658 657 509 639 561 552 596 600 648 579 553 580 650 441 232 457 680 608 514 570 576 602 609 666 678 635 504 667 615 462 619 334 323 654 609 631 600 569 657 668 672 683 658 670 684 665 552 675 626 706 709 509 703 716 671 710 649 647 711 616 685 584 668 681 654 727 672 717 700 711 696 603 652 680 639 662 673 701 659 737 621 706 736 655 712 683 515 644 717 473 660 739 642 666 732 742 649 665 621 497 679 410 530 650 718 703 720 705 652 675 342 646 681 727 723 679 749 752 741 748 628 766 759 720 691 693 698 745 646 644 709 689 697 578 781 463 780 695 772 760 735 677 773 775 728 774 751 749 612 696 794 383 738 763 792 782 697 806 686 441 753 785 799 775 802 689 788 754 782 796 811 576 804 626 756 786 758 823 804 716 790 797 824 611 828 615 537 763 731 788 690 778 785 813 624 796 784 805 798 816 831 802 840 701 784 791 653 531 767 526 714 821 839 738 661 839 847 809 817 704 810 813 843 843 445 686 849 704 732 768 766 833 793 765 848 767 853 814 780 810 645 830 855 806 825 700 827 756 837 855 854 799 529 854 841 661 819 786 736 691 842 735 817 861 742 853 715 708 692 816 870 861 878 851 820 818 859 851 340 879 530 831 838 867 760 835 876 874 849 875 391 871 885 852 872 863 645 712 790 828 809 618 844 859 844 888 693 827 860 807 886 894 889 886 891 764 568 825 751 874 835 897 893 573 875 574 754 715 794 895 866 584 573 857 737 881 830 899 895 765 837 909 847 774 884 887 869 893 887 866 824 892 903 892 882 910 807 867 918 902 858 889 856 805 908 922 921 728 901 898 929 878 692 865 924 881 917 933 898 925 852 932 901 944 923 812 925 940 936 945 912 922 884 940 897 908 916 902 899 938 937 938 714 944 865 951 917 860 950 927 936 953 914 957 924 945 950 948 958 949 885 913 914 818 932 910 941 905 937 939 848 959 841 955 948 933 890 862 961 970 964 962 972 923 960 951 920 962 928 957 958 973 954 964 942 832 978 979 928 954 903 953 977 916 969 984 985 979 963 894 965 955 970 967 986 978 764 982 982 967 986 990 989 990 988 987 988 966 963 888 983 984 993 966 994 973 977 991 996 985 975 969 965 959 997 974 997 891 983 909 993 992 989 991 974 975 996 998 995 998 987
0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1 0 1
1549 1884
1309 1698
1254 1437
1457 1488
1636 1662
