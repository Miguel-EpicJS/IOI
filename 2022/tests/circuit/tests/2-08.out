4faa78c22ee35083e49b65601698a1bd98a87a0f
OK
846777934
543886020
117265458
170290282
281705356
